harbitech.sql

```diff
@@ -88,14 +88,16 @@ FOREIGN KEY(question_id) REFERENCES Question(id));
 
 SELECT * FROM Category;
 SELECT * FROM Subcategory;
-SELECT * FROM Course; 
+SELECT * FROM Course; 	
 SELECT * FROM Explanation;
 SELECT * FROM Section;
 SELECT * FROM Activity;
 SELECT * FROM Video;
 SELECT * FROM Question;
 SELECT * FROM Alternative;
 
+UPDATE Course SET visibility = 'PRIVATE' WHERE id = 1
+
```

Comment:
Precisa colocar um `;` senão o script não é executado mais.

```suggestion
UPDATE Course SET visibility = 'PRIVATE' WHERE id = 1;
```

Inclusive, seria bom ter esse script de queries separado dos de DLL (`CREATE TABLE`).

Reply:
Obrigada pela dica, vou colocar junto com o data.sql